activesupport/test/core_ext/time_ext_test.rb

```diff
@@ -454,6 +454,9 @@ def test_offset_change
     assert_equal 10, Time.new(2005, 2, 22, 15, 15, 0, "-08:00").change(nsec: 10).nsec
     assert_raise(ArgumentError) { Time.new(2005, 2, 22, 15, 15, 45, "-08:00").change(usec: 1000000) }
     assert_raise(ArgumentError) { Time.new(2005, 2, 22, 15, 15, 45, "-08:00").change(nsec: 1000000000) }
+    with_env_tz "GB" do
```

Comment:
```suggestion
    with_env_tz "BST" do
```

for the record, this makes the test pass.

Reply:
I've tried a bunch of other things but couldn't find any other way to get it passing. Two questions

1) Why `GB` as the time zone? Where did that come from?
2) I think in general you should use `Time.zone.parse` for this sort of thing - any reason you're not here?

Reply:
1. I believe `GB` is what should be there, based on others not being timezones, `US/Eastern` and `NZ` etc. It might pass with `BST` as its not a valid region, so defaults to another region which does leave daylight savings time at the same time as the `GB` region, but thats just me guessing. All the `with_env_tz "GB"` does is sets `ENV['TZ'] = "GB"`, so I think it has to be one of these codes: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

2. We only noticed the issue during the hour that the clocks went back in the UK, in the production code we were using we had `Time.now.beginning_of_minute`, but given `Time#beginning_of_minute` just calls `change(sec: 0)` the problem is not actually with `Time#beginning_of_minute`. Using `Time.zone` was the only way to recreate it in a test. I had managed to also recreate it using Timecop but I was not going to introduce that here.

Reply:
The reason I'm suggesting maybe it should be `"BST"` is because that's what `Time.parse(...).zone` is returning. But I'm not really sure.

FWIW in our production apps we don't set `ENV["TZ"]` at all, and we use https://www.rubydoc.info/gems/rubocop/0.57.2/RuboCop/Cop/Rails/TimeZone to remind people to use `Time.zone.now...` instead of `Time.now...`.

All of which is to say, I'm not really sure if this is a bug or not just yet.